eng/Version.Details.xml

```diff
@@ -10,151 +10,197 @@
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.EventSource" Version="6.0.0-alpha.1.20557.2">
+    <Dependency Name="Microsoft.Extensions.Logging.EventSource" Version="6.0.0-alpha.1.20575.3">
       <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="6.0.0-alpha.1.20557.2">
+    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="6.0.0-alpha.1.20575.3">
       <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-alpha.1.20557.2">
+    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-alpha.1.20575.3">
       <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0-alpha.1.20557.2">
+    <Dependency Name="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0-alpha.1.20575.3">
       <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.0-alpha.1.20557.2">
+    <Dependency Name="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.0-alpha.1.20575.3">
       <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options" Version="6.0.0-alpha.1.20557.2">
+    <Dependency Name="Microsoft.Extensions.Options" Version="6.0.0-alpha.1.20575.3">
       <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.0-alpha.1.20557.2">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.0-alpha.1.20575.3">
       <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
     </Dependency>
     <!--
          Win-x64 is used here because we have picked an arbitrary runtime identifier to flow the version of the latest NETCore.App runtime.
          All Runtime.$rid packages should have the same version.
     -->
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.0-alpha.1.20557.2">
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.0-alpha.1.20575.3">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>6cfef20c1524b8d4c1495a652dfe9e898c5a486b</Sha>
+      <Sha>e08358f5a2450430fd871a955e1d2c380a08c093</Sha>
     </Dependency>
     <Dependency Name="Microsoft.NETCore.App.Internal" Version="6.0.0-alpha.1.20557.2">
```

Comment:
@mmitche do you happen to know if this package stopped being produced / updated? The bor did not update it while updating other versions.